packages/tables/demo/table.stories.mdx

```diff
@@ -46,6 +46,7 @@ import { TABLE_DATA as DATA } from './stories/data';
       isReadOnly: { control: 'boolean' },
       hasOverflow: { control: 'boolean', table: { category: 'Cell' } },
       isTruncated: { control: 'boolean', table: { category: 'Cell' } },
+      isHidden: { control: 'boolean', table: { category: 'Cell' } },
```

Comment:
```suggestion
      isHidden: { control: 'boolean', name: 'hidden', table: { category: 'Cell' } },
```

...similar to [Label](https://github.com/zendeskgarden/react-components/blob/33efd5e18dd8a390466601fd2423f0c705f7f7e9/packages/forms/demo/stories/common.tsx#L14).

Reply:
💥 [c33a847](https://github.com/zendeskgarden/react-components/pull/1514/commits/c33a847918a15f751b0ce425c5ae60c93ef3d771)